api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionSignPaths.scala

```diff
@@ -85,8 +93,58 @@ trait SubmissionSignPaths
 
     onComplete(fSubmission) {
       case Success(sub) =>
+        if (sub.status.code == 10) {
+          emailSignature(supervisor, sub)
+        }
         complete(ToResponseMarshallable(Receipt(sub.end, sub.receipt, sub.status)))
       case Failure(error) => completeWithInternalError(uri, error)
     }
   }
+
+  private def emailSignature(supervisor: ActorRef, submission: Submission)(implicit ec: ExecutionContext) = {
+    val emails = findEmailsById(submission.id.institutionId)
+    emails.map(emailSeq => {
+      emailSeq.foreach(t => {
+        val username = t._1 + " " + t._2
+        sendMail(t._3, username, submission)
+      })
+    })
+  }
+
+  private def sendMail(address: String, username: String, submission: Submission) = {
+    val config = ConfigFactory.load()
+    val host = config.getString("hmda.mail.host")
+    val port = config.getString("hmda.mail.port")
+
+    val properties = System.getProperties
+    properties.put("mail.smtp.host", host)
+    properties.put("mail.smtp.port", port)
+
+    val session = Session.getDefaultInstance(properties)
+    val message = new MimeMessage(session)
+
+    val date = getFormattedDate
+
+    val text = s"$username,\n\nCongratulations, you've completed filing your HMDA data for filing period ${submission.id.period}.\n" +
```

Comment:
Should we include the institution name in the email text? Seems like that would be helpful for users who represent multiple institutions. @keileyo

Reply:
This would be a nice to have, I think. What is LOE?

Reply:
It's not difficult.  Could you provide an updated email template with the institution name included?